src/algebra/category/Module/monoidal.lean

```diff
@@ -44,26 +44,27 @@ open_locale tensor_product
 local attribute [ext] tensor_product.ext
 
 /-- (implementation) tensor product of R-modules -/
-def tensor_obj (M N : Module R) : Module R := Module.of R (M ⊗[R] N)
+def tensor_obj (M N : Module.{v} R) : Module R := Module.of R (M ⊗[R] N)
```

Comment:
Isn't this removing the ability to take the tensor product of two vectors spaces in different universes?

Reply:
Indeed, you're right. I'll try to actually fix it later today. OTOH, the "public" `monoidal_category` API not only assumes that `M` and `N` live in the same universe, but also assumes that this is the same universe where `R` lives.

Reply:
I'm not strongly opposed to the universe weakening, it's just not something I'm qualified to make a decision on here

Reply:
Let's see if the new version (without universe restrictions) compiles.